pkg/kusttest/kusttestharness.go

```diff
@@ -141,6 +141,23 @@ func (th *KustTestHarness) LoadAndRunTransformer(
 	return resMap
 }
 
+func (th *KustTestHarness) ErrorFromLoadAndRunTransformer(
```

Comment:
no need for this

Reply:
I need this function to test the transformer with error returned.
The existing `LoadAndRunTransformer` doesn't expect error happens.

Reply:
ah, right this is a transformer whose failure is expected.
OK, factor out the common code please.